testing/environments/docker/kibana/Dockerfile

```diff
@@ -8,7 +8,7 @@ EXPOSE 5601
 ARG DOWNLOAD_URL=https://snapshots.elastic.co/downloads
 ARG ELASTIC_VERSION=7.0.0-alpha1-SNAPSHOT
 ARG CACHE_BUST=1
-ARG XPACK=1
```

Comment:
Is this change intentional. I think this will affect our default testing.

Reply:
see `testing/environments/docker/elasticsearch/Dockerfile`
```
ARG ELASTIC_VERSION
ARG DOWNLOAD_URL
ARG ES_JAVA_OPTS
ARG CACHE_BUST=1
ARG XPACK
```

The XPACK value for Elasticsearch and Kibana should be set by `testing/environments/snapshot.yml` and `testing/environments/snapshot-noxpack.yml`  


Shall I create a separate PR for this issue?

Reply:
I created https://github.com/elastic/beats/pull/6383/